workers-site/index.js

```diff
@@ -43,19 +43,24 @@ async function handleEvent(event) {
     }
     return await getAssetFromKV(event, options)
   } catch (e) {
-    if (DEBUG) {
-      return new Response(e.message || e.toString(), {
-        status: 404,
-      })
-    } else {
-      let requestedAsset = options.mapRequestToAsset
-        ? options.mapRequestToAsset(event.request).url
-        : mapRequestToAsset(event.request.url)
-
-      return new Response(`"${requestedAsset}" not found`, {
-        status: 404,
-      })
+    if (!DEBUG) {
+      try {
+        return await getAssetFromKV(event, {
+          mapRequestToAsset: notFoundAsset(),
```

Comment:
```suggestion
          mapRequestToAsset: (req) => new Request(`${new URL (event.request.url).origin}/404.html` )
```

Reply:
and then delete the function because it's confusing

Reply:
my thought was to give a pattern for customizing this

Reply:
pretty will make it several lines

Reply:
prettier makes it one line and deletes the parens lol

Reply:
done!